hyperstart.go

```diff
@@ -437,6 +481,10 @@ func (h *hyper) startPod(pod Pod) error {
 		}
 	}
 
+	if err := h.addDrives(&pod); err != nil {
```

Comment:
Do you still pass the mounted devicemapper mount pass via 9pfs? I did not see code to disable that? Did I miss it. I am worried about block device corruption if we continue to pass it in via 9p. Even thing like accidental atime update

Also is there a easy way to toggle the behaviour. i.e, switch back to 9pfs even for devicemapper. Offering this option may be required if there are users who may care about the lack of support for dynamic file copy in/out of the container via 

```
docker cp [OPTIONS] CONTAINER:SRC_PATH DEST_PATH|-
docker cp [OPTIONS] SRC_PATH|- CONTAINER:DEST_PATH
```

Also we need to document this limitation with the direct assignment approach.